src/api/spec/factories/linked_project.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :linked_project do
+    sequence(:position) { |n| n }
+    project
+
+    trait :local do
+      linked_db_project { association :project }
+    end
+
+    trait :remote do
+      linked_remote_project_name { 'openSUSE.org:home:hennevogel:myfirstproject' }
+    end
```

Comment:
I would still define them, otherwise it's a bit hard to get meaning into this factory...

Reply:
> I would still define them, otherwise it's a bit hard to get meaning into this factory...

I would say this is an issue inherited from the definition of the model. In other words, defining `trait`s in a factory doesn't improve by itself a better use of the factory, IMHO.

In the other hand, comments would clarify the use of the factory. We could create comments in the model or here in the factory.

Reply:
Yeah the whole association naming make your head spin...

Can you maybe push a commit how you imagine the comments that are more clear than the traits? :)

Reply:
> Can you maybe push a commit how you imagine the comments that are more clear than the traits? :)

The text of the comments would be the same as the traits. Creating comments instead of creating traits (which are not going to be used) will by itself make things clearer. If I define traits, I would expect them to be used somewhere. If I create comments, no other futher action is expected.

Basically, I find annoying creating traits which are not going to be used in any other spec. If we want to clarify what are some fields of a factory or a model for, we already can make use of comments.

Reply:
In case we ever need some clarification on the fields used by the `linked_project` factory, it would be something similar to the commit _"Clarify fields used in linked_project factory"_ I have just added. Feel free to modify it to fit your needs, in case you find those lines not clear enough.